src/components/useList/components/ListContainerView/ListContainerView.scss

```diff
@@ -0,0 +1,17 @@
+@use '../../../variables';
+
+$block: '.#{variables.$ns}list-container-view';
+
+#{$block} {
+    box-sizing: border-box;
+    width: 100%;
+    outline: none;
+
+    &_fixed-height {
+        height: var(--g-list-height, 300px);
```

Comment:
Let's not mix stable and unstable feautures together, `--g-list-height` probably will be a part of CSS for old List